src/NBitcoin/InMemoryNoSqlRepository.cs

```diff
@@ -8,20 +8,25 @@ public class InMemoryNoSqlRepository : NoSqlRepository
     {
         Dictionary<string, byte[]> _Table = new Dictionary<string, byte[]>();
```

Comment:
Again, whilst we are here, rename to `table`.

Reply:
Done